### ISSUE TITLE
Lat/long looks flipped by pyproj.Transformer

### ISSUE THREAD

**Issue by miraanand** [nasaharvest/presto#6]
Hi, this is really interesting! I'm trying out the downstream task notebook as I am looking into applying this approach to wetland classification.

This repo uses pyproj's `Transformer.transform()` ([eg](https://github.com/nasaharvest/presto/blob/main/downstream_task_demo.ipynb?short_path=4621113#LL166C3-L166C22) in the downstream task notebook) which can sometimes [swap the order of coordinates](https://proj.org/faq.html#why-is-the-axis-ordering-in-proj-not-consistent) depending on the coordinate system. Specifically, EPSG:4326 uses lat/long whereas the input UTM data uses long/lat. In the example I linked, this results in the two variables being switched.

Adding `always_xy=True` to the [transformer](https://github.com/nasaharvest/presto/blob/main/downstream_task_demo.ipynb?short_path=4621113#L159) would keep the ordering consistent throughout (see the warning on the [docs](https://pyproj4.github.io/pyproj/stable/api/transformer.html#transformer)).

It's not clear to me whether this would affect the pre-trained model, but I see the same line in at least the [eurosat](https://github.com/nasaharvest/presto/blob/a8e3db77fe37aa2f8b231cc2aeca317313690764/presto/eval/eurosat_eval.py#L105) and [treesat](https://github.com/nasaharvest/presto/blob/a8e3db77fe37aa2f8b231cc2aeca317313690764/presto/eval/treesat_eval.py#L146) eval files. It doesn't seem to significantly affect the F1 score on the example notebook however. 

Even if there aren't other impacts I think that updating the order so that the variables are named correctly would avoid confusion.

Thanks!

**Comment by gabrieltseng** [comment]
Hi @miraanand ,

Thank you for your interest in the project, and for spotting this. Fortunately, it doesn't affect the pre-trained model, since [no transformation of the coordinates](https://github.com/nasaharvest/presto/blob/a8e3db77fe37aa2f8b231cc2aeca317313690764/presto/dataops/pipelines/ee_pipeline.py#L50) is necessary for the pretraining data.

However, you are right that this affects the EuroSat and TreeSatAI tasks. I have rerun everything with the corrected latlons (results are in #7 ), and implemented the `always_xy` fix you suggested.

Thanks again!
